src/vaadin-text-field-mixin.html

```diff
@@ -808,11 +846,15 @@
       }
     }
 
-    /** @private */
-    _getActiveErrorId(invalid, errorMessage, errorId) {
-      this._setOrToggleAttribute('aria-describedby',
-        (errorMessage && invalid ? errorId : undefined),
-        this.focusElement);
+    _getActiveErrorId(invalid, errorMessage, errorId, helperText, helperTextId, hasSlottedHelper) {
+      const ids = [];
+      if (helperText || hasSlottedHelper) {
+        ids.push(helperTextId);
+      }
+      if (errorMessage && invalid) {
+        ids.push(errorId);
+      }
+      this._setOrToggleAttribute('aria-describedby', ids.join(' '), this.focusElement);
```

Comment:
Did we test this change with actual screen readers? If not, let's create a separate issue for it.

Reply:
I am not sure about it. I gave a quick test here, and it looks like it's effective.